auslib/test/web/test_client.py

```diff
@@ -731,6 +758,16 @@ def testVersion6GetWithDistribution(self):
         <patch type="complete" URL="http://a.com/y" hashFunction="sha512" hashValue="13" size="12"/>
     </update>
 </updates>
+""")
+
+    def testVersion6GetWithDistributionInList(self):
```

Comment:
Can you add a few more tests? Eg: to make sure substring matching isn't happening, and that a totally different string doesn't match?